samples/iothub-connect-to-eventhubs/IotHubConnection.cs

```diff
@@ -23,6 +24,9 @@ namespace IotHubToEventHubsSample
     ///
     public static class IotHubConnection
     {
+        /// <summary>The regular expression used to parse the Event Hub name from the IoT Hub redirection address.</summary>
+        private static readonly Regex EventHubNameExpression = new Regex(@":\d+\/(.*)\/\$management", RegexOptions.IgnoreCase | RegexOptions.Compiled | RegexOptions.CultureInvariant);
```

Comment:
@chradek :  These changes were the result of our conversation today.   I gently tweaked the expression that you're using to allow for any port.

Reply:
That looks reasonable to me.